daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -1817,7 +1817,11 @@ private[lf] object SBuiltin {
     }
   }
 
-  /** $error :: Text -> a */
+  /** $error :: Text -> a
+    *
+    * This builtin is reserved for LF prior to 1.14. For LF >= 1.14, a catchable DAML
+    * exception should be used instead.
```

Comment:
:cry: 
In fact what I told you is not true, `SBError` have not been deprecated when introducing exceptions. 
It is still used by the compiler in some corner cases.

Reply:
Sorry.

Reply:
No problem. I'll revert these comment changes.